### ISSUE TITLE
YouTube throttling?

### ISSUE THREAD

**Issue by JWhiteford** [futo-org/grayjay-android#361]
When I open up grayjay every platform loads almost instantly except YouTube. Is this the same throttling that is doing to Firefox?

**Comment by koen-futo** [comment]
Are there any others with this issue? Just polled within the team and nobody seems to be able to reproduce this. Could be a new player version?

**Comment by koen-futo** [comment]
Is it only the initial load?

**Comment by JWhiteford** [comment]
I've only see it on initial load.

**Comment by Kelvin-FUTO** [comment]
You're talking about the home pages correct? Not video playback?

Youtube home is always a bit sluggish (in browser). Other platforms just have faster home pages from my experience.
If you could elaborate more on what exactly is slow and how slow that would help.

Its very unlikely its related to the Firefox stuff. We do not run Youtube's javascript for home pages etc.

**Comment by muflihansari** [comment]
I have experienced slow thing too. I only use YouTube plugin so in my case Grayjay is replacement of YouTube app however, it loads very slowly. The app seems heavy. It takes too long to load at first. Even after that, it is not smooth at all. Changing the quality of video is confusing, just make it simpler like YouTube. Turning captions on or off seems to make screen black for 2 seconds. Double tapping for a quick 10-second forward or backward also slows it down. It's so slow. I was embarrassed when I tried to show my friends. My friends said they would watch a 5 second ad instead of this much long wait.

By the app, I mean YouTube in the app because I only use YouTube in it.
